config/initializers/okcomputer.rb

```diff
@@ -8,9 +8,9 @@
 OkComputer::Registry.register 'feature-resque-down', OkComputer::ResqueDownCheck.new
 
 # check for backed up resque queues: this is a low volume app, so the threshold is low
-Resque.queues.each do |queue|
-  OkComputer::Registry.register "feature-#{queue}-queue-depth", OkComputer::ResqueBackedUpCheck.new(queue, 5)
-end
+# Resque.queues.each do |queue|
```

Comment:
?

Reply:
Oh, this shouldn't be in this pr, good catch.